e2e/terraform/gke/README.md

```diff
@@ -0,0 +1,39 @@
+# STEPS TO CREATE GKE CLUSTER USING TERRAFORM
+
+## Step-1 : Obtain the Google Credentials from Google Developers Console 
+
+- Log into the Google Developers Console and select a project.
+- The API Manager view should be selected, click on "Credentials" on the left, then "Create credentials," and finally "Service account key."
+- Select "Compute Engine default service account" in the "Service account" dropdown, and select "JSON" as the key type.
+- Clicking "Create" will download your credentials.
+- Rename this file to account.json and place in the directory where the terraform templates are present/will-be-placed
+
+## Step-2 : Obtain the Google project ID
+
+- Log into the Google Developers Console to be sent to the Google API library page screen
+- Click the Select a project drop-down in the upper left
+- Copy the Project ID for the desired project from the window that shows 
```

Comment:
Where is the Project ID used later?

Reply:
We will use it in the variables.tf file. Added a step in the README alluding to this.